src/clients/test.py

```diff
@@ -7,8 +7,8 @@ class TestClient:
     def __init__(self, account: Account):
         self.account = account
 
-    def schedule(self, image_path, caption):
-        print(f"Received in schedule args: {image_path}, {caption}")
+    def schedule(self, image_path, caption, content_tags):
```

Comment:
If they aren't used here yet, I prefer keeping it out

Reply:
ok, i threw it out from unused places.